utils/utils.go

```diff
@@ -77,13 +81,14 @@ func CommandLog(t *testing.T, stdout string, stderr string, err error) {
 		}
 		if stderr != "" {
 			if err != nil {
+				// fatal error, non-zero exit
 				t.Error(stderr)
```

Comment:
How about:
```
t.Fatal(stderr)
```

Reply:
Yes, that was a mistake.

Reply:
Actually it was correct. It should only log the error. Fatal will exit it right away which isn't what we want here.